python/lancedb/remote/table.py

```diff
@@ -98,6 +98,8 @@ def search(
         return LanceVectorQueryBuilder(self, query, vector_column_name)
 
     def _execute_query(self, query: Query) -> pa.Table:
+        if query.is_prefilter:
+            raise NotImplementedError("Cloud support for prefiltering is coming soon")
```

Comment:
Related, if you want to take a look: https://github.com/lancedb/lancedb/pull/519

Reply:
yeah, we can remove this once the new thing is deployed. Had feedback from users that the gap between Cloud and OSS in features is confusing.